<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22,67,75);
color: rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane {
    border-top: 2px solid rgb(255, 255, 255);
}

QTabWidget::tab-bar {
    left: 5px;
}

QTabBar::tab {
    background: rgb(147,204,213);
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 13ex;
    padding: 3px;
	font: 75 11pt &quot;Source Code Pro&quot; white;
}

QTabBar::tab:hover {
    background: rgb(72, 160, 177);
}

QTabBar::tab:selected {
	background: rgb(72, 160, 177);
}

QTabBar::tab:!selected {
    margin-top: 2px;
}

QTabBar::tab:selected {
    margin-left: -4px;
    margin-right: -4px;
}

QTabBar::tab:first:selected {
    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
    margin: 0; /* if there is only one tab, we don't want overlapping margins */
}


</string>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTreeView" name="dirView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QTreeView {
    background-color: rgb(255, 255, 255);
	color: rgb(0,0,0);
}

QTreeView {
    show-decoration-selected: 1;
}

QTreeView::item {
    border: 1px solid rgb(147, 204, 213);
    border-top-color: transparent;
    border-bottom-color: transparent;
}

QTreeView::item:hover {
    background: rgb(147, 204, 213);
    border: 1px solid rgba(68, 151, 167, 144);
}

QTreeView::item:selected {
    border: rgb(72, 160, 177);
}

QTreeView::item:selected:active{
    background: rgb(72, 160, 177);
	border: 1px solid rgba(72, 160, 177, 144);
}
QTreeView{selection-background-color: rgba(72, 160, 177, 0);}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTabWidget" name="outputWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane {
    border-top: 2px solid rgb(255, 255, 255);
}

QTabWidget::tab-bar {
    left: 5px;
}

QTabBar::tab {
    background: rgb(147,204,213);
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 20ex;
    padding: 3px;
	font: 75 12pt &quot;Source Code Pro&quot;;
}

QTabBar::tab:hover {
    background: rgb(72, 160, 177);
}

QTabBar::tab:selected {
	background: rgb(72, 160, 177);
}

QTabBar::tab:!selected {
    margin-top: 2px;
}

QTabBar::tab:selected {
    margin-left: -4px;
    margin-right: -4px;
}

QTabBar::tab:first:selected {
    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
    margin: 0; /* if there is only one tab, we don't want overlapping margins */
}


</string>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 160, 177);</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 160, 177);</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_SCAPES"/>
    <addaction name="actionInstruction_Book"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 160, 177);</string>
    </property>
    <property name="title">
     <string>Compile</string>
    </property>
    <addaction name="actionCompile"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 160, 177);</string>
    </property>
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCompile"/>
   <addaction name="menuRun"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 166, 174);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCompile"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/imgs/interface/resource/new_icon.png</normaloff>:/imgs/interface/resource/new_icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/imgs/interface/resource/open_icon.png</normaloff>:/imgs/interface/resource/open_icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/imgs/interface/resource/save_icon.png</normaloff>:/imgs/interface/resource/save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout_SCAPES">
   <property name="text">
    <string>About SCAPES</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionInstruction_Book">
   <property name="text">
    <string>Instruction Book</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/imgs/interface/resource/exit_icon.png</normaloff>:/imgs/interface/resource/exit_icon.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/imgs/interface/resource/compile_icon.png</normaloff>:/imgs/interface/resource/compile_icon.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/imgs/interface/resource/run_icon.png</normaloff>:/imgs/interface/resource/run_icon.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>:/imgs/interface/resource/saveas_icon.png</normaloff>:/imgs/interface/resource/saveas_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
